art/estimators/estimator.py

```diff
@@ -246,7 +294,7 @@ def _apply_preprocessing(self, x, y, fit: bool) -> Tuple[Any, Any]:
         :rtype: Format as expected by the `model`
         """
         if self.preprocessing:
```

Comment:
Should this be `if self.preprocessing_operations`?

Reply:
Yes, that's correct! Thank you.